nirc_ehr/resources/schemas/dbscripts/postgresql/nirc_ehr-24.001-24.002.sql

```diff
@@ -0,0 +1 @@
+SELECT core.executeJavaUpgradeCode('reloadFolder');
```

Comment:
Please also add a line to import from alias_category.tsv:

SELECT core.executeJavaUpgradeCode('importFromTsv;ehr_lookups;death_reason;/data/alias_category.tsv');

Reply:
done